src/test/scala/org/constellation/domain/redownload/RedownloadServiceTest.scala

```diff
@@ -308,10 +316,12 @@ class RedownloadServiceTest
       val persistSecond = redownloadService.persistCreatedSnapshot(4L, "bbbb", SortedMap.empty)
       val check = redownloadService.getCreatedSnapshots()
 
-      (persistFirst >> persistSecond >> check).unsafeRunSync shouldBe Map(
-        2L -> PersistedSnapshotProposal("aaaa", SortedMap.empty),
-        4L -> PersistedSnapshotProposal("bbbb", SortedMap.empty)
-      )
+      (persistFirst >> persistSecond >> check).unsafeRunSync.toList should matchPattern {
+        case List(
+        (2L, Signed(_, SnapshotProposal("aaaa", 2L, _))),
+        (4L, Signed(_, SnapshotProposal("bbbb", 4L, _)))
```

Comment:
indent?

Reply:
It's a part of the pattern, so likely it shouldn't be indented. scalafmt also doesn't want to indent it, I can do it only manually.